dbt/include/snowflake/macros/materializations/dynamic_table/ddl.sql

```diff
@@ -1,46 +1,96 @@
-{% macro snowflake__get_alter_dynamic_table_as_sql(
```

Comment:
Is this file a bunch of helper DDL for the snowflake methods for building out this table? If I'm right, would love to see that documented at filetop. If I'm wrong, another reason to document our intentions 😁

Reply:
Yes, it's the former. It's the pieces that go into `dynamic_table/materialization.sql`. When the materialization is in `dbt-core`, this boils down to just a file `dynamic_table.sql`. I broke it out here to emphasize that the materialization is local (and because it's a lot of code with slightly different uses). Agreed, I should (and will) add a doc block at the top.